conf/pxe_cluster/ks.cfg

```diff
@@ -32,21 +32,27 @@ text
 #Install OS instead of upgrade
 install
 url --url http://172.16.141.17/ubuntu/
+
+#Disk partitioning information
+part /boot/efi --fstype=efi --grow --maxsize=200 --size=20
+part / --fstype ext4 --size 1 --grow --asprimary
+
 # System bootloader configuration
 bootloader --location=mbr
 
 zerombr yes
 #Partition clearing information
 clearpart --all --initlabel
-#Disk partitioning information
-part / --fstype ext4 --size 1 --grow --asprimary
+
 #Skip question(s) about not having swap partition or overwriting the existing FS
 preseed partman-lvm/device_remove_lvm boolean true
 preseed partman/confirm_write_new_label boolean true
 preseed partman/confirm boolean true
 preseed partman/confirm_nooverwrite boolean true
 preseed --owner partman-basicfilesystems partman-basicfilesystems/no_swap boolean false.
 
+network --bootproto=dhcp --device=en0
```

Comment:
The device en0 probably needs to be configurable based on hosts.yaml info.

Reply:
It is.  However, currently, the host.yaml and code assumes the if names are the same for all boxes, so it uses the list_ifface for the subnet to replace en0 with whatever